aws/cloudformation/components/www_redirect.yml.erb

```diff
@@ -81,6 +75,33 @@
           QueryStringsConfig: 
             QueryStringBehavior: all
 
+  <%=app%>RedirectFunction:
+    Type: AWS::CloudFront::Function
+    Properties: 
+      Name: www_redirect
+      FunctionConfig: 
+        Comment: Redirects a subdomain to the root domain
+        Runtime: cloudfront-js-1.0
+      AutoPublish: true
+      FunctionCode: !Sub |
+        function handler(event) {
+          var newUrl = 'https://<%=CDO.pegasus_hostname%>';
+
+          // Append path
+          newUrl += event.request.uri;
+
+          // Append query string
+          // - skipping because it's annoying to parse.
```

Comment:
stray comments?

```suggestion
```

Reply:
I wasn't sure if it was worth acknowledging that we were not including query params.